### PR TITLE
Fix system logic in prune unused images dialog

### LINKED ISSUE #1022: Pruning unused images doesn't work as root
When using cockpit-podman as root, "Prune unused images" always results in this error for me:

    The endpoint must be either a unix path or port number

I think this happens because we try to prune user images, but that doesn't work for root.  The Browser console shows:

    $XDG_RUNTIME_DIR is not present. Cannot use user service.

### PR DESCRIPTION
**Pull request by jelly**
Check if the user or system service is avaiable instead of checking if
both are true. Which does not work as userServiceAvailbe is null when
logging in as root.

Closes #1022

### CODE REVIEW

**Comment by marusak** [comment]
Can you please also add test where you reproduce #1022 ?

File: src/PruneUnusedImagesModal.jsx [hunk 1]
```diff
@@ -52,10 +52,9 @@ function ImageOptions({ images, checked, isSystem, handleChange, name, showCheck
 class PruneUnusedImagesModal extends React.Component {
     constructor(props) {
         super(props);
-        const isSystem = this.props.userServiceAvailable && this.props.systemServiceAvailable;
         this.state = {
-            deleteUserImages: true,
-            deleteSystemImages: isSystem,
+            deleteUserImages: this.props.userServiceAvailable !== null && this.props.userServiceAvailable,
```
Comment (marusak) [inline src/PruneUnusedImagesModal.jsx:56]: Couldn't this just be `deleteUserImages: this.props.userServiceAvailable`?

**Review by marusak — APPROVED** [review]
Thank you! Just one question, mostly OOI

**Comment by martinpitt** [comment]
> jelle | pitti: I kinda like the explicit null check, it being null was what bit us
> jelle | pitti: we had null && bool  which resuls to null :D

**Pull request merged**